workers/main/src/services/OAuth2/IOAuth2TokenManager.ts

```diff
@@ -0,0 +1,21 @@
+import { TokenData } from './types';
+
+export interface TokenStorageProvider {
+  save(tokenData: TokenData): Promise<void>;
+
+  load(): TokenData | null;
+
+  clear(): Promise<void>;
+}
+
+export interface TokenRefreshProvider {
```

Comment:
Let's name interfaces consistently: `IOAuth2TokenManager` and `ITokenRefreshProvider` or `OAuth2TokenManager` and `TokenRefreshProvider`. Also add eslint rule that will prevent AI from creating new issues like this. (As next task)

Reply:
interfaces "I" prefix has been removed to follow modern TypeScript conventions